src/components/badges/ShareToTwitterIfNeeded.tsx

```diff
@@ -60,7 +61,9 @@ function ShareToTwitterIfNeededSuspended() {
           className={margin('ml-auto', 'xs:ml-0')}
           onClick={closeNotification}
         >
-          <Cross />
+          <div className={width('w-6')}>
+            <Cross />
+          </div>
```

Comment:
See it the second time. Should we create a separate component? Or move `<div className={width('w-6')}>` inside `<Cross />`?